### ISSUE TITLE
Font files not copied to Android assets folder if they are already correctly noted in the link-assets-manifest.json file

### ISSUE THREAD

**Issue by teststation5** [unimonkiez/react-native-asset#59]
in a react-native project where gitignore is purposely ignoring all files in the android\app\src\main\assets folder, but the link-assets-manifest.json file is in source control.  Someone building the project on a new machine will use the react-native-asset command line but the asset files will not be copied to the android\app\src\main\assets folder because the link-assets-manifest.json file is up to date. The assets will not be available on the Android device and it can take many, many hours to figure out why.

Suggested improvement would be to check both if the asset file exists in the link-assets-manifest.json file and also if it physically exists in the android\app\src\main\assets folder and copy it there if not.

**Comment by 1uss1** [comment]
I agree with your suggestion. I encountered the same issue in my project and my solution for it was to add the link-assets-manifest.json files into gitignore.
